mongo-init.js

```diff
@@ -1,7 +1,12 @@
 // Create Databases
 const dbPims = db.getSiblingDB('pims');
 const dbRemsAdmin = db.getSiblingDB('remsadmin');
-
+dbRemsAdmin.createUser({ user: "rems-user",
+  pwd: "pass",
```

Comment:
instead of creating a default user here, we should use the default admin authentication that comes built in with mongo. This will make it easier to use the system with any mongo server in production as it does not require and preexisting data to be configured